operatorconfig/driverconfig/common/k8s-1.22-values.yaml

```diff
@@ -2,26 +2,26 @@
 images:
   # "images.attacher" defines the container images used for the csi attacher
   # container.
```

Comment:
Please revert the changes in this file as we are not supporting K8s v1.22

Reply:
taken care